cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -792,7 +792,9 @@ public void deleteQueues(HttpRequest request, HttpResponder responder) {
   @Path("/streams")
   public void deleteStreams(HttpRequest request, HttpResponder responder) {
     try {
-      streamAdmin.dropAll();
+      //TODO: move this to v3 namespaced-stream handler when delete is implemented for streams
+      String namespace = getAuthenticatedAccountId(request);
```

Comment:
Can we use ``Constants.DEFAULT_NAMESPACE`` here directly? ``getAuthenticatedAccountId`` doesn't do anything but return that constant. Until we figure out what to do with authentication, better to not use that method?